### PR TITLE
update drawing tool help

### LINKED ISSUE #3579: Project builder help text in workflow editor for drawing tools using markdown that doesn't render
Presuming the ** should be 
`<b></b>.` 
screen shot 2017-03-09 at 11 54 28 pm

If I'm correct, I can make that change. cc @eatyourgreens

### PR DESCRIPTION
**Pull request by mschwamb**
add missing help descriptions of publicly available drawing tools. Also switched markdown bold characters to html bold tags

Fixes #3579

### CODE REVIEW

File: app/classifier/tasks/generic-editor.cjsx [hunk 1]
```diff
@@ -204,15 +204,17 @@ module.exports = React.createClass
...
-                <small className="form-help">*point:* X marks the spot.</small><br />
-                <small className="form-help">*line:* a straight line at any angle.</small><br />
-                <small className="form-help">*polygon:* an arbitrary shape made of point-to-point lines.</small><br />
-                <small className="form-help">*rectangle:* a box of any size and length-width ratio; this tool *cannot* be rotated.</small><br />
-                <small className="form-help">*circle:* a point and a radius.</small><br />
-                <small className="form-help">*ellipse:* an oval of any size and axis ratio; this tool *can* be rotated.</small><br />
-                <small className="form-help">*column rectangle:* a box with full height but variable width; this tool *cannot* be rotated.</small>
+                <small className="form-help"><b>bezier:</b> an arbitrary shape made of point-to-point curves. The midpoint of each segment drawn can dragged to adjust the curvature. </small><br />
```
Comment (simensta) [inline app/classifier/tasks/generic-editor.cjsx]: small typo - I think you want "can be dragged to adjust the curvature"

File: app/classifier/tasks/generic-editor.cjsx [hunk 2]
```diff
@@ -204,15 +204,17 @@ module.exports = React.createClass
...
+                <small className="form-help"><b>circle:</b> a point and a radius.</small><br />
+                <small className="form-help"><b>column:</b> a box with full height but variable width; this tool <b>cannot</b> be rotated.</small>
```
Comment (simensta) [inline app/classifier/tasks/generic-editor.cjsx]: missing the `<br />` tag for the column tool

File: app/classifier/tasks/generic-editor.cjsx [hunk 3]
```diff
@@ -204,15 +204,17 @@ module.exports = React.createClass
...
+                <small className="form-help"><b>ellipse:</b> an oval of any size and axis ratio; this tool <b>can</b> be rotated.</small><br />
+                <small className="form-help"><b>line:</b>: a straight line at any angle.</small><br />
+                <small className="form-help"><b>point:</b>: X marks the spot.</small><br />
```
Comment (simensta) [inline app/classifier/tasks/generic-editor.cjsx]: There are double colons after the `line` and `point` titles.

**Comment by mschwamb** [comment]
@simensta  I think that fixes all my typos (sorry for not catching those earlier) and fixes the italic styling.  Let me know if there are other changes needed.

**Comment by simensta** [comment]
Looks good! Thanks @mschwamb!

**Comment by mschwamb** [comment]
Thanks @simensta

**Pull request merged**